proto/beacon/p2p/v1/types.proto

```diff
@@ -79,11 +79,19 @@ message CandidatePoWReceiptRootRecord {
   uint64 votes = 2;
 }
 
+// PendingAttestationRecord is deprecated entirely by AttestationRecord.
```

Comment:
Let me confirm that this is true. I don't see why this is a different data structure.

Reply:
We should have 2. One is `AggregatedAttestation` which represents `AttestationRecord` in spec
Another one is `PendingAttestationRecord`
`AggregatedAttestation` tracks attestations before processing. `PendingAttestationRecord` tracks attestations post processing